### PR TITLE
Adds `brew docs` to close #13834

### LINKED ISSUE #13834: `brew docs` as an alias for `open  https://docs.brew.sh`
### Provide a detailed description of the proposed feature

`brew docs` as a shortcut for `open https://docs.brew.sh`

Also udpate brew help

```diff
$ brew help
(....)
Further help:
  brew commands
  brew help [COMMAND]
  man brew
-  https://docs.brew.sh
+ brew docs
```

### What is the motivation for the feature?

Laziness is one of the three great virtues of the programmer, as defined by Larry Wall.

Laziness is the quality that makes you go to great effort to reduce overall energy expenditure. It makes you write labor-saving programs that other people will find useful and document what you wrote so you don't have to answer so many questions about it.

### How will the feature be relevant to at least 90% of Homebrew users?

_Benefits for users_

It makes it marginally easier to 90% of Homebrew users to access the docs at https://docs.brew.sh 

And maybe find the information they need, which is nice.

_Benefits for maintainers_

It is marginally nicer for Homebrew maintainers that Homebrew users are empowered to find the documentation they need. They would have to repeat themselves marginally less.

Of course, many users still won't read the docs, but now maintainers can reply 

> This is documented in `$ brew docs`, section `Common Issues`

This kind of reply makes it marginally more likely that users read the docs next time. 
Yes it's just a RTFM, but I think/hope it's an empowering "Read the Friendly Manual" RTFM, not the "Read the Fucking Manual" RTFM.

_Benefits for documentation writers_

As someone who writes technical documentation myself, I know this is hard work and I know that I like it when more users actually use and benefit from the documentation.

### What alternatives to the feature have been considered?

Bookmarking https://docs.brew.sh

But I'm too lazy to bookmark things and use browser bookmarks correctly.

### PR DESCRIPTION
**Pull request by troymccabe**
- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [X] Have you added an explanation of what your changes do and why you'd like us to include them?
- [X] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [X] Have you successfully run `brew style` with your changes locally?
- [X] Have you successfully run `brew typecheck` with your changes locally?
- [X] Have you successfully run `brew tests` with your changes locally?

-----

Per the discussion in #13834, here's an MVP for opening docs in a browser. 

Fixes #13834

### CODE REVIEW

File: Library/Homebrew/cmd/docs.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  sig { returns(CLI::Parser) }
+  def docs_args
+    Homebrew::CLI::Parser.new do
+      description <<~EOS
+        Open Homebrew's docs (https://docs.brew.sh) in a browser.
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/docs.rb:15]:
```suggestion
        Open Homebrew's online documentation (https://docs.brew.sh) in a browser.
```

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @troymccabe!

File: Library/Homebrew/cmd/docs.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  sig { returns(CLI::Parser) }
+  def docs_args
+    Homebrew::CLI::Parser.new do
+      description <<~EOS
+        Open Homebrew's online documentation (https://docs.brew.sh) in a browser.
```
Comment (carlocab) [inline Library/Homebrew/cmd/docs.rb:15]:
```suggestion
        Open Homebrew's online documentation (#{HOMEBREW_DOCS_WWW}) in a browser.
```

**Comment by jmfayard** [comment]
🎉 Thanks for implementing my issue @troymccabe!

**Pull request merged**